README.md

```diff
@@ -1,16 +1,20 @@
 FirebaseUI: Simplyify Mobile Development with Firebase
```

Comment:
This typo isn't yours, but can you correct it anyway? `Simplyify` -> `Simplify` 😅

Also, FirebaseUI is now beyond mobile, so maybe this whole header should be

>FirebaseUI: Simplifying App Development with Firebase